scripts/update-whitelist.ts

```diff
@@ -0,0 +1,72 @@
+// add and remove must be comma delimited strings
+//-----------------------------------------------
+
+import { SigningCosmWasmClient } from '@cosmjs/cosmwasm-stargate';
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+import { calculateFee, coins, GasPrice } from '@cosmjs/stargate';
+
+const config = require('./config');
+
+async function updateWhitelist(add: string, remove: string) {
+  const add_addresses = add == '' ? null : add.split(',');
+  const remove_addresses = remove == '' ? null : remove.split(',');
+  if (add_addresses != null) {
+    console.log('add addresses: ', add_addresses.join(','));
```

Comment:
Convert these to stars addresses here. I already told creators they can collect any Cosmos address for whitelists. Maybe break https://github.com/public-awesome/stargaze-nft/blob/main/scripts/mint.ts#L66 into a util function.